src/cript/nodes/core.py

```diff
@@ -296,6 +300,26 @@ class ReturnTuple:
         previous_condense_to_uuid = copy.deepcopy(NodeEncoder.condense_to_uuid)
         NodeEncoder.condense_to_uuid = condense_to_uuid
 
+        # For validation of just a node and its direct children, we enable a new mode in Node encoding
+        # For this node we explicitly list the nodes that are fully expressed, every other node is a UUID
+        previous_only_not_uuid: Optional[list[str]] = NodeEncoder.only_not_uuid
+        if only_this_node_full:
```

Comment:
if this is to separate just the direct child nodes, then maybe it should be in its own function/method and gets called from here, goes in there, separates out what it needs, and returns the data to here

Reply:
no, I don't think that is a good idea.
It makes the logic convulted and complicated.